driver/server/grpc.go

```diff
@@ -58,32 +58,56 @@ func newDriverManifest(manifest *manifest.Manifest) protocol1.DriverManifest {
 	}
 }
 
+// SupportedLanguages implements protocol1.Service.
 func (s service) SupportedLanguages(_ *protocol1.SupportedLanguagesRequest) *protocol1.SupportedLanguagesResponse {
-	m, _ := s.d.Manifest()
-	return &protocol1.SupportedLanguagesResponse{Languages: []protocol1.DriverManifest{
-		newDriverManifest(&m),
-	}}
+	ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
```

Comment:
This is pre-existing so we shouldn't try to fix it now, but we should think about making these methods accept a context as an argument. For now, I suggest using `context.TODO()` in each of these cases—it gives the same value as `context.Background()` but the name makes it easier to find cases that need propagation up the stack.

Reply:
It's not worth to add an argument since https://github.com/bblfsh/bblfshd/pull/210 will remove v1 support eventually. The "fix" was added only to make sure the `Languages` call won't block forever. Thus the TODO part is not actionable, so I've set it to `Background` until we remove v1 completely.